### DISCUSSION TITLE
Truncated terminal window on initial Codespaces start

### DISCUSSION THREAD

**Discussion by valhuber** [community/community#32790] [Codespaces] [answered]
Minor issue -- when I start Codespaces on [this project](https://github.com/ApiLogicServer/ApiLogicProject), I see this:

![garbled teminal](https://user-images.githubusercontent.com/5579631/189693387-a3155040-6d0b-433c-af00-3901fb6aa6c6.png)

It's fine if I open a new terminal window.

**Comment by dmilburn** [comment, marked as answer]
Hi @valhuber, thanks for reporting!

Were you using safari? If so, I believe it'll be fixed in this next [vscode](https://github.com/microsoft/vscode/pull/160332) release by: https://github.com/microsoft/vscode/issues/160070

**Reply by valhuber** [reply]
yes, confirmed was fine in a different Browser... thanks for getting back!
